mobile/test/common/integration/rtds_integration_test.cc

```diff
@@ -73,6 +73,23 @@ TEST_P(RtdsIntegrationTest, RtdsReload) {
 
   // Verify that the Runtime config values are from the RTDS response.
   EXPECT_TRUE(Runtime::runtimeFeatureEnabled("envoy.reloadable_features.test_feature_false"));
+
```

Comment:
Why was this test added to the mobile integration test, and not the non-mobile one?

Reply:
I picked this because this is only sotw, we can add both since there is no any `Resource` integration tests right now.